LoopKit/LoopAlgorithm/LoopAlgorithmInput.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LoopAlgorithmInput.swift
+//  LoopKit
+//
+//  Created by Pete Schwamb on 9/21/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public enum DoseRecommendationType: String {
+    case manualBolus
+    case automaticBolus
+    case tempBasal
```

Comment:
Should `tempBasal` be `automaticTempBasal`? It seems we will be adding manual temp basal in the not to distant future. Do you think there will be a dose recommendation with manual temp basal?